src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskProvidingConfiguration.swift

```diff
@@ -21,6 +21,18 @@ import Foundation
 
 /// Used to configure a `RiskLevelProvider`.
 struct RiskProvidingConfiguration {
+
+    static let defaultExposureDetectionsInterval = 24 / defaultMaxExposureDetectionsPerInterval
+    private static let defaultMaxExposureDetectionsPerInterval = 1
+
+	static var defaultConfiguration: RiskProvidingConfiguration {
```

Comment:
_Swiftier_ name:
```suggestion
	static var default: RiskProvidingConfiguration {
```

If accepted: needs also adjustments to code above!

Reply:
I would also prefer "default", but its a keyword.
This works "`default`" ☺️

Reply:
🤦‍♂️ true…